cmd/simplerequester/util.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	insJose "github.com/insolar/go-jose"
```

Comment:
why alias here

Reply:
because we used 2 go-jose libraries, standart squire implementation for P256 and insolar implementation for P-256K curve